seqr/management/tests/check_bam_cram_paths_tests.py

```diff
@@ -3,32 +3,43 @@
 from django.core.management import call_command
 from django.test import TestCase
 
+from seqr.models import IgvSample
 
+
+@mock.patch('hail.hadoop_is_file', lambda path: False)
 class CheckBamCramPathsTest(TestCase):
     fixtures = ['users', '1kg_project']
 
+    def setUp(self):
+        existing_sample = IgvSample.objects.first()
+        IgvSample.objects.create(
```

Comment:
Why not just update the existing igv sample to have gs file path so you can use it, or even add a new fixture. Setting up test data like this is is a real last resort and an antipattern

Reply:
I think the db should start with 1 record that needs to be updated and 1 that doesn't, and to make sure it's only modifying the right one. I agree adding hundreds of records this way would be suboptimal, but adding 1 or 2 seems ok, unless there's a better way to add unittest-specific records.

Reply:
I don't see any harm in adding a record to the fixtures file so we can start with 2 records in the configuration we want